cli/portforward.go

```diff
@@ -92,6 +92,7 @@ func (r *RootCmd) portForward() *clibase.Cmd {
 				Fetch: func(ctx context.Context) (codersdk.WorkspaceAgent, error) {
 					return client.WorkspaceAgent(ctx, workspaceAgent.ID)
 				},
+				Wait: false,
```

Comment:
Review: Before change from `NoWait -> Wait`, these were bugs, added the explicit false here to signal intent.